### DISCUSSION TITLE
random name alias generator

### DISCUSSION THREAD

**Discussion by emhl** [anonaddy/anonaddy#673] [Feature Requests]
it would be awesome if there would be an option that are of the firstname.lastname variant. optionally with varying seperator caracters and an optional number at the end. so a lot like with the random caracters option but, looking a lot more like the email adresses normal people would create on a freemail service or get from their work

**Comment by willbrowningme** [comment]
Thanks for the suggestion, that is a good idea that I'll definitely try to add.
